app/poros/book.rb

```diff
@@ -7,12 +7,12 @@ class Book
               :rating,
               :genre
   def initialize(data)
-    @title = data[:data][:attributes][:title]
-    @author = data[:data][:attributes][:authors][0]
-    @summary = data[:data][:attributes][:description]
-    # @reviews = data[:data][:attributes][:reviews]
+    @title = data[:attributes][:title]
+    @author = data[:attributes][:authors][0]
+    @summary = data[:attributes][:description]
+    # @reviews = data[:attributes][:reviews]
```

Comment:
same issue here: do we want to leave commented out code to remind us of stretch ? or should we remove this because we're pushing to main?